src/protocols/rdp/settings.c

```diff
@@ -1770,6 +1773,9 @@ void guac_rdp_push_settings(guac_client* client,
     /* Explicitly set flag value */
     rdp_settings->PerformanceFlags = guac_rdp_get_performance_flags(guac_settings);
 
+    /* Set explicit connection type to LAN to prevent auto-detection */
+    freerdp_settings_set_uint32(rdp_settings, FreeRDP_ConnectionType, CONNECTION_TYPE_LAN);
```

Comment:
Is this valid when `HAVE_SETTERS_GETTERS` is not defined?

Reply:
Confirmed - `freerdp_settings_set_uint32()` is present since FreeRDP 2.0.0, as are the `FreeRDP_ConnectionType` and `CONNECTION_TYPE_LAN` constants.